spring-integration-jdbc/src/test/java/org/springframework/integration/jdbc/postgres/PostgresChannelMessageTableSubscriberTests.java

```diff
@@ -72,7 +73,7 @@
  * @author Igor Lovich
  * @author Adama Sorho
  * @author Johannes Edmeier
- *
+ * @author Norbert Schneider
  * @since 6.0
```

Comment:
I prefer a blank line after the `@author` section to emphasize on them

Reply:
Changed.